sdk/src/main/scala/io/horizen/account/api/rpc/types/TxPoolInspect.scala

```diff
@@ -0,0 +1,14 @@
+package io.horizen.account.api.rpc.types
+
+import com.fasterxml.jackson.annotation.JsonView
+import io.horizen.evm.Address
+import io.horizen.json.Views
+
+import java.math.BigInteger
+import scala.collection.concurrent.TrieMap
+import scala.collection.mutable
+
+@JsonView(Array(classOf[Views.Default]))
+class TxPoolInspect(
+    val pending: TrieMap[Address, mutable.SortedMap[BigInteger, String]],
```

Comment:
be sure that the map is serialzied like `563662: "0x958c1fa64b34db746925c6f8a3dd81128e40355e: 1051546810000000000 wei + 90000 gas × 20000000000 wei",`
without additional object `{}`

Reply:
checked, it serialized the response without additional object `{}`
The nonce is serialized as string with decimal representation